test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -11007,14 +11007,16 @@ def add_server_dyn_res(self, custom_resource, script_body=None,
                          configuration settings.
         :type validate: bool
         """
+        super_user_grp = PbsGroup(grp.getgrgid(0).gr_name, gid=0) 
```

Comment:
should we be making this change in ptl/utils/pbs_testsuite.py generically?

Reply:
Actually, pbs_testsuite does have this lined up nicely as "ROOT_USER" but unfortunately, pbs_testlib has no access to it. I didn't want to complicate it by making it visible to pbs_testlib so I just replicated the code

Reply:
Bhroam and I had some more discussion about it and we thought that passing a string would be better. I've changed it to a string now.